site/components/IconsSearch.tsx

```diff
@@ -0,0 +1,25 @@
+import { Grid, Input } from "@mui/material";
+import React from "react";
```

Comment:
Don't import **React** completely. you are using `ChangeEventHandler` type. import it like this:
```import type { ChangeEventHandler } from 'react'```